NitroxServer-Subnautica/App.config

```diff
@@ -16,7 +16,7 @@
         <add key="StartHealth" value="80" />
         <add key="StartFood" value="50.5" />
         <add key="StartWater" value="90.5" />
-        <add key="StartInfection" value="0" />
+        <add key="StartInfection" value="0.1" />
```

Comment:
Let's add a comment to both of these specifying why 0.1 is required.

Reply:
Left a comment at the second location as the App.config cannot have comments in it.